core/store/orm/orm.go

```diff
@@ -71,7 +71,9 @@ func NewORM(uri string, timeout models.Duration, shutdownSignal gracefulpanic.Si
 		advisoryLockTimeout: timeout,
 		shutdownSignal:      shutdownSignal,
 	}
-	orm.MustEnsureAdvisoryLock()
+	if err = orm.MustEnsureAdvisoryLock(); err != nil {
+		return nil, err
```

Comment:
Wait if NewORM returns an error for the standby nodes it looks like we'll just shut down the app [https://github.com/smartcontractkit/chainlink/blob/f105cafd45788ef26aa35cec3298b93dffa00929/core/store/store.go#L88] ?

Reply:
(which it always would return an error since is the standby node so itd timeout getting the lock)

Reply:
Yes this is correct, and it always worked this way